conf/modules.config

```
@@ -676,7 +715,7 @@ process {
 
     withName: AMP_DATABASE_DOWNLOAD {
         publishDir = [
-            path: { "${params.outdir}/databases/${params.amp_ampcombi_db}" },
+            path: { "${params.outdir}/databases/" },
```

```suggestion
            path: { "${params.outdir}/databases/ampcombi" },
```

If we use the interproscan example above?